service/s3/s3manager/download.go

```diff
@@ -50,6 +62,9 @@ type Downloader struct {
 	// List of request options that will be passed down to individual API
 	// operation requests made by the downloader.
 	RequestOptions []request.Option
+
+	// Defines the buffer strategy used when downloading a part
+	BufferProvider WriterReadFromProvider
```

Comment:
would it be helpful to describe this member and its usage within the Downloader more?